src/main/java/com/salesforce/storm/spout/dynamic/filter/StaticMessageFilter.java

```diff
@@ -38,10 +38,9 @@
     /**
      * We need a way to make this instance unique from others, so we use a UUID.
      */
-    private final String id;
+    private String id = UUID.randomUUID().toString();
```

Comment:
why not final?

Reply:
Something weird was happening when it deserialized when I set it inside of the constructor, I think it has to do with the way GSON reflects, but I'm not sure honestly. This worked around it.  This reminds me... this class should be moved into src/test.